templates/bin/setup

```diff
@@ -1,37 +1,37 @@
 #!/usr/bin/env ruby
```

Comment:
To me it does not make a lot of sense to have this file here. We need to update it at every Rails change and seems a bit stupid. Why don't we remove it?

Reply:
Perfectly fine.